editor/src/clj/editor/tile_source.clj

```diff
@@ -1024,7 +1088,8 @@
         inner-padding :inner-padding
         sprite-trim-mode :sprite-trim-mode))))
 
-(def ^:private default-animation
+(def ^:private new-animation-defaults
+  "Default field values for added Tile$Animation instances."
```

Comment:
Renamed this and added a clarifying comment since we now also have `protobuf-animation-defaults`.